### PR TITLE
[TimeSeriesInsights] Fix operation groups and tags across all operations.

### PR DESCRIPTION
**Pull request by yeskarthik**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json [hunk 1]
```diff
@@ -45,7 +45,10 @@
   "paths": {
     "/availability": {
       "get": {
-        "operationId": "GetAvailability",
+        "tags": [
+          "TimeSeries"
+        ],
+        "operationId": "TimeSeries_AvailabilityGet",
```
Comment (dmdenmsft) [inline specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json]:
```suggestion
        "operationId": "TimeSeries_GetAvailability",
```
In both English generated docs and generated code, "Get Something" and GetSomething() is more common, consistent and more readable (especially that in other places we do put verb first).

File: specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json [hunk 2]
```diff
@@ -98,7 +101,10 @@
     },
     "/eventSchema": {
       "post": {
-        "operationId": "GetEventSchema",
+        "tags": [
+          "TimeSeries"
+        ],
+        "operationId": "TimeSeries_EventSchemaGet",
```
Comment (dmdenmsft) [inline specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json]:
```suggestion
        "operationId": "TimeSeries_GetEventSchema",
```

File: specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json [hunk 3]
```diff
@@ -545,7 +569,10 @@
     },
     "/timeseries/instances/search": {
       "post": {
-        "operationId": "SearchInstancesPaged",
+        "tags": [
+          "Instances"
+        ],
+        "operationId": "Instances_SearchPaged",
```
Comment (dmdenmsft) [inline specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json]: I am divided on adding Paged or not - adding it makes it more precise and consistent, on the other hand, Search will never be non-paged. I would probably remove Paged - makes it simpler.
```suggestion
        "operationId": "Instances_Search",
```

Comment (dmdenmsft) [reply specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json]: Or should we drop Paged everywhere?

File: specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json [hunk 4]
```diff
@@ -1666,11 +1705,11 @@
         "modelSettings": {
           "description": "Model settings including model name, Time Series ID properties and default type ID.",
           "readOnly": true,
-          "$ref": "#/definitions/ModelSettings"
+          "$ref": "#/definitions/TimeSeriesModelSettings"
```
Comment (dmdenmsft) [inline specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json]: I would keep original. We don't add TimeSeries to other data structures... or does it collide during code gen? Then I think the change is right.

**Review by dmdenmsft — COMMENTED** [review]
Thanks for doing this!

File: specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json (hunk 4 above)
Comment (yeskarthik) [reply specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/preview/2018-11-01-preview/timeseriesinsights.json]: Yes, It collides with the ModelSettings_ operation group.

**Review by tjprescott — APPROVED** [review]

**Pull request merged**